api-guidelines/global/json/canonical-data-types/rules/should-use-common-money-object.md

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
Can you please add the "otto" part to the file name so that the file name matches the title? :)